src/jit/importer.cpp

```diff
@@ -6989,17 +6989,23 @@ var_types Compiler::impImportJitTestLabelMark(int numArgs)
     return node->TypeGet();
 }
 #endif // DEBUG
-/*****************************************************************************
-   For struct return types either adjust the return type to an enregisterable
-   type, or set the struct return flag.
- */
-
-GenTreePtr                Compiler::impFixupStructReturn(GenTreePtr     call,
-                                                         CORINFO_CLASS_HANDLE retClsHnd)
+//---------------------------------------------------------------------------
+//  For a call node that returns a struct type either adjust the return type
```

Comment:
CC: Please use the function header from the Jit  Coding Convention document. It should start with:
//------------------------------------------------------------------------
// Function name: Short description of the function
//